test/run-test.ps1

```diff
@@ -131,6 +131,15 @@ Get-ChildItem -Recurse -Filter Dockerfile |
                     }
                 }
 
+                if ($sdkTag -like "*2.0-sdk") {
+                    # Temporary workaround https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/dogfooding.md#option-2-self-contained
+                    exec { docker run --rm `
+                        -v ${selfContainedVol}":${containerRoot}volume" `
+                        $selfContainedImage `
+                        chmod u+x ${containerRoot}volume${platformDirSeparator}test
```

Comment:
Why does this need to be done inside a Docker container?  Couldn't we just run chmod on `${selfContainedVol}${dirSeparator}test`?